src/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,54 @@
+import configureStore from "redux-mock-store";
+import TestRoute from "components/Routes/TestRoute";
+import { mount } from "enzyme";
+import React from "react";
+import { Provider } from "react-redux";
+import { MemoryRouter } from "react-router-dom";
+import dataDump from "testing/complete-redux-store-dump";
+
+import Breadcrumb from "./Breadcrumb";
+
+const mockStore = configureStore([]);
+
+describe("Breadcrumb", () => {
+  it("displays correctly on model details", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/models/eggman@external/group-test"]}>
+          <TestRoute path="/models/:userName/:modelName?">
+            <Breadcrumb />
+          </TestRoute>
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("[data-test='breadcrumb-application']").length).toBe(0);
+    expect(wrapper.find("[data-test='breadcrumb-model']").text()).toStrictEqual(
+      "group-test"
+    );
+  });
+
+  it("displays correctly on application details", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={["/models/eggman@external/group-test/app/easyrsa"]}
+        >
+          <TestRoute path="/models/:userName/:modelName?/app/:appName?">
+            <Breadcrumb />
+          </TestRoute>
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("[data-test='breadcrumb-model']").text()).toStrictEqual(
```

Comment:
I was expecting this test to check for something like "group-test > Applications > easyrsa" instead of grabbing each element individually. Can you move the `find` up to `p-breadcrumbs__items` to achieve this?

Reply:
Not sure a test to check the order is correct is that valuable but I have added it.